### DISCUSSION TITLE
How do I use it together with Vue fire app check custom provider?

### DISCUSSION THREAD

**Discussion by artrayd** [nuxt-modules/turnstile#325] [Q&A]
### 📚 Is your documentation request related to a problem?

I want to use turnstile together with app check. 

For app check I'm using it:
https://vuefire.vuejs.org/guide/app-check.html

Here is my nuxt.config code:
    appCheck: {
      debug: process.env.NODE_ENV !== 'production',
      isTokenAutoRefreshEnabled: true,
      provider: turnstile,
      key: 'my-key',
    },

**Comment by dargmuesli** [comment]
What exactly are you currently limited by?

**Comment by artrayd** [comment]
@dargmuesli I don't understand how to configure it at nux.config.js

**Reply by dargmuesli** [reply]
You would need to specify which step of the readme / documentation does not let you understand the setup or you'd need to provide a error message for others to be able to help you.
